fastlane/Fastfile

```diff
@@ -194,7 +197,7 @@ platform :ios do
   # bundle exec fastlane new_hotfix_release version:10.6.1
   # bundle exec fastlane new_hotfix_release skip_confirm:true version:10.6.1
   #####################################################################################
-  desc 'Creates a new hotfix branch from the given tag'
+  desc 'Creates a new hotfix branch for the given version:x.y.z. The branch will be cut from the tag x.y of the previous release'
```

Comment:
Not sure if it's worth clarifying here, but if we were to do an hotfix on top of another hotfix, e.g. what I did to test https://github.com/wordpress-mobile/release-toolkit/pull/287, the branch would be cut from the tag of the previous hotfix release, which is `x.y.n` instead of `x.y`.

🤷‍♂️

Reply:
Yeah I thought about that after the fact. Not sure how to express this in a wording that doesn't end up being too complicated trying to explain all the subtle cases 🤔

Reply:
Yeah. It's a minutia. Maybe we should spend all this mental energy in making the code and tests so good that we will never need to ship more than one hotfix 🤣 

---

As I mentioned in another PR where I made this comment, this is good for me. 

![ship it](https://media.giphy.com/media/79qf1N4RJtc8o/giphy.gif)